schema/queue/event/impl/MqEventSender.go

```diff
@@ -46,13 +47,12 @@ type CommonEventSender struct {
 func (sender *CommonEventSender) Send(batch *p_buff.EventBatch) error {
 
 	if batch == nil {
-		sender.Logger.Fatal().Msg("Value for send can't be null")
+		sender.Logger.Error().Msg("Value for send can't be null")
+		return fmt.Errorf("null value for sending")
```

Comment:
I think we should create constant error for the file instead of creating it every time